File: package/android/src/oldarch/WorkletsSpec.java
package com.worklets;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;
import com.facebook.react.bridge.Promise;

abstract class WorkletsSpec extends ReactContextBaseJavaModule {
  WorkletsSpec(ReactApplicationContext context) {
    super(context);
  }

  public abstract boolean install();
}

File: package/android/src/oldarch/WorkletsSpec.kt
package com.margelo.worklets

import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.bridge.ReactContextBaseJavaModule
import com.facebook.react.bridge.Promise

abstract class WorkletsSpec internal constructor(context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
  abstract fun install(): String?
}
